server/index.js

```diff
@@ -27,15 +27,12 @@ app.use('/api/v1/', repayments);
 
 app.use('/docs-api', swaggerUi.serve, swaggerUi.setup(swaggerSpec));
 
+app.use((req, res) => {
```

Comment:
I would like to know how you intend to catch routing errors with this. Also, I would love to know the reasoning behind this implementation.

Reply:
I am still awaiting your response